tests/test_http_request.py

```diff
@@ -1281,6 +1281,16 @@ class JsonRequestTest(RequestTest):
         b'Accept': [b'application/json, text/javascript, */*; q=0.01'],
     }
 
+    def test_replace_custom(self):
+        """Test Request.replace() method"""
```

Comment:
To keep the format of the tests written by @jenniferlindberg, "# GROUP 12 ADDED TEST CASE" should be added as comment to the implemented test.